src/style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
+}
+
+.stars--0 .stars_star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars_star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars_star:nth-child(n + 2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars_star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars_star:nth-child(n + 3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 .stars_star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars_star:nth-child(n + 4) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 .stars_star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars_star:nth-child(n + 5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars_star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars .stars_star:first-child {
+  margin-right: 2px;
+  margin-left: 0;
+}
+
+.stars .stars_star:last-child {
+  margin-left: 2px;
+  margin-right: 0;
+}
```

Comment:
add margin to one side instead of both

Reply:
but I have global styles for each element to add margin on both sides, so a have to override those styles with margin only on one side.
If I would've add only 4px margin-right to one side I still had to override the last element not to have it

Reply:
I guess I understood how it could be done